File: fizzbuzz/src/perl/fizzbuzz.pl
use strict;
use warnings;
for my $n (1..50) {
    if ($n % 3 == 0 && $n % 5 == 0) {
        print "fizz buzz\n";
    }
    elsif ($n % 3 == 0) {
        print "fizz\n";
    }
    elsif ($n % 5 == 0) {
        print "buzz\n";
    }
    else {
        print "$n\n";
    }
}

File: src/program/fsharp/fizzbuzz/fizzbuzz.fs
for n = 1 to 50 do
    match n % 3, n % 5 with
    | 0, 0 -> printfn "fizz buzz"
    | 0, _ -> printfn "fizz"
    | _, 0 -> printfn "buzz"
    | _, _ -> printfn "%d" n
done
